Definition of move (#1073)
thewca/wca-regulations: wca-regulations.md

        - A3b1) When the judge believes the competitor is ready, the judge asks "READY?". The competitor must be ready to start the attempt within one minute from the moment they are asked, or else the competitor forfeits the attempt (DNS), at the discretion of the judge.
        - A3b2) The competitor starts the attempt by confirming that they are ready, and the judge uncovers the puzzle and starts timing the inspection.
    - A3c) The competitor may pick up the puzzle during inspection.

>         - A3c1) The competitor must not apply moves nor intentional alignments during inspection. Penalty: disqualification of the attempt (DNF).

Reviewer: ```suggestion
        - A3c1) The competitor must not apply moves nor intentional alignments (see [Regulation A3c2](regulations:regulation:A3c2)) during inspection. Penalty: disqualification of the attempt (DNF).
```

>         - A3c1) The competitor must not apply moves nor intentional changes in alignment during inspection. Penalty: disqualification of the attempt (DNF). Exception: see [Regulation A3c2](regulations:regulation:A3c2).

Reviewer 2: Nit: we don't use "nor" in the Regulations.
```suggestion
        - A3c1) The competitor must not apply moves or intentional changes in alignment during inspection. Penalty: disqualification of the attempt (DNF). Exception: see [Regulation A3c2](regulations:regulation:A3c2).
```

        - A3b2) The competitor starts the attempt by confirming that they are ready, and the judge uncovers the puzzle and starts timing the inspection.
    - A3c) The competitor may pick up the puzzle during inspection.
        - A3c1) The competitor must not apply moves or intentional changes in alignment during inspection. Penalty: disqualification of the attempt (DNF). Exception: see [Regulation A3c2](regulations:regulation:A3c2) and [Regulation A3c5](regulations:regulation:A3c5).
        - A3c2) If the parts of the puzzle are not fully aligned when the cover is removed, then the competitor may align the faces, as long as misalignments stay within the limits of [Regulation 10f](regulations:regulation:10f).
        - A3c3) The competitor may reset the timer before the start of the solve.
        - A3c4) For Square-1, if the use of a thin object inside the puzzle has been enforced by the organization team (see [Regulation A2b1](regulations:regulation:A2b1)), the competitor may remove the object from the puzzle during inspection.

>         - A3c5) For Square-1, the competitor should not be penalized for any accidental move made within the limits of [Regulation 10f](regulations:regulation:10f).

Reviewer: I think this one is already covered by another PR that has already been merged (it has been written as an exception, so we can remove this one I think)

@Nanush7 
```suggestion
```

Author: That's the problem. As draft was ahead of this branch, I had to resolve the conflict. And this is probably the best way. Otherwise, we would have something like: "Exception: see reg (...). For Sq-1 ..."

Reviewer: oh yeah this makes more sense

    - A6b) The competitor is responsible for stopping the Stackmat timer correctly.
        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.

>     - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves or any change in alignment have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.

Reviewer 2: Nit: subject-verb agreeement.
```suggestion
    - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no move or any change in alignment have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
```

Reviewer: ```suggestion
    - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no move or any change in alignment has been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
```

@lgarron would this make more sense?

>     - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves, nor any change in alignment, have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.

Reviewer: ```suggestion
    - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves or any change in alignment have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
```

    - A6d) The competitor must stop the timer using both hands, placed flat on the sensors with palms down. Penalty: time penalty (+2 seconds).

>     - A6e) After stopping the timer, the competitor must not touch or apply a move nor any change in alignment to the puzzle, until the judge has inspected the puzzle.

Reviewer: ```suggestion
    - A6e) After stopping the timer, the competitor must not touch, apply a move, or make any change in alignment to the puzzle, until the judge has inspected the puzzle.
```

- B3) Memorization phase:
    - B3a) The competitor may pick up the puzzle during the memorization phase.
    - B3b) The competitor must not make physical notes. Penalty: disqualification of the attempt (DNF).

>     - B3c) The competitor must not apply moves nor intentional alignments to the puzzle during the memorization phase. Penalty: disqualification of the attempt (DNF).

Reviewer: ```suggestion
    - B3c) The competitor must not apply moves nor intentional alignments (see [Regulation A3c2](regulations:regulation:A3c2)) to the puzzle during the memorization phase. Penalty: disqualification of the attempt (DNF).
```

>     - B3c) The competitor must not apply moves nor intentional changes in alignment (see [Regulation A3c2](regulations:regulation:A3c2)) to the puzzle during the memorization phase. Penalty: disqualification of the attempt (DNF).

Reviewer 2: ```suggestion
    - B3c) The competitor must not apply moves or intentional changes in alignment (see [Regulation A3c2](regulations:regulation:A3c2)) to the puzzle during the memorization phase. Penalty: disqualification of the attempt (DNF).
```

    - B3d) If the parts of the puzzle are not fully aligned, then the competitor may align the faces, as long as misalignments stay within the limits of [Regulation 10f](regulations:regulation:10f) (see [Regulation A3c2](regulations:regulation:A3c2)).
- B4) Blindfolded phase:
    - B4a) The competitor dons the blindfold to start the blindfolded phase.
        - B4a1) Until the competitor applies the first move to the puzzle, they may remove the blindfold to return to the memorization phase.

>     - B4b) The competitor must not apply moves nor intentional changes in alignment to the puzzle before they have fully donned the blindfold. Penalty: disqualification of the attempt (DNF).

Reviewer 2: ```suggestion
    - B4b) The competitor must not apply moves or intentional changes in alignment to the puzzle before they have fully donned the blindfold. Penalty: disqualification of the attempt (DNF).
```
